corehq/apps/export/models/new.py

```diff
@@ -2378,16 +2406,17 @@ def _generate_schema_from_case_property_mapping(cls, case_property_mapping, pare
                 group_schema.items.append(ScalarItem(
                     path=[PathNode(name=prop)],
                     label=prop,
-                    last_occurrences={app_id: app_version},
+                    last_occurrences={app_id: app_version} if app_id else {},
                 ))
 
-        for case_type, identifier in parent_types:
-            group_schema.items.append(CaseIndexItem(
-                path=[PathNode(name='indices'), PathNode(name=case_type)],
-                label='{}.{}'.format(identifier, case_type),
-                last_occurrences={app_id: app_version},
-                tag=PROPERTY_TAG_CASE,
-            ))
+        if parent_types:
```

Comment:
will support for this be added later?

Reply:
You mean via data dictionary?
I don't know of any plans to have case relationship set in DD.

Does this block users from doing data export for related cases? That seems concerning.

Reply:
You can't export related case data but you can export the case indices. I don't know how often this is used but if DD can't support this then I'd say it's a non-starter unless we can get this info from somewhere else.

Reply:
Okay.

parent_types are fetched from apps when iterating apps [here](https://github.com/dimagi/commcare-hq/blob/581731f8d00bf034bddc3d593e145b31bf4ee692/corehq/apps/export/models/new.py#L2375).
We could support it by iterating through apps though that is what we are trying to avoid.

@esoergel I guess this detail was not available when you wrote the CEP.
Any thoughts on this?

Reply:
thanks for catching that @snopoke

Reply:
It would generally be useful to include case relationships in the data dictionary, though I haven't thought much further about it than that.  It's also noteworthy that case indices created via save-to-case [don't appear in exports currently](https://dimagi.atlassian.net/browse/USH-1993), which could perhaps be addressed by storing those relationships in the DD.  This was something I'd thought of in [the Future of Data Dictionary doc](https://docs.google.com/document/d/1uDmtxwz6qXZgrpISseDNX0nFACRzSttrOvmJ8UxBaow/edit), but I hadn't put together the pieces to see that this would be a blocker for exports.